internal/controller/integration_test.go

```diff
@@ -1457,13 +1458,27 @@ var _ = Describe("Integration Test", func() {
 			profileCombinedMGTMDT.Data.LustreStorage.CombinedMGTMDT = true
 			Expect(createNnfStorageProfile(profileExternalMGS, true)).ToNot(BeNil())
 			Expect(createNnfStorageProfile(profileCombinedMGTMDT, true)).ToNot(BeNil())
+
+			nnfLustreMgt = &nnfv1alpha1.NnfLustreMGT{
+				TypeMeta: metav1.TypeMeta{},
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "profile-mgs",
+					Namespace: corev1.NamespaceDefault,
+				},
+				Spec: nnfv1alpha1.NnfLustreMGTSpec{
+					Addresses:   []string{profileMgsNid},
+					FsNameStart: "dddddddd",
+				},
+			}
+			Expect(k8sClient.Create(context.TODO(), nnfLustreMgt)).To(Succeed())
```

Comment:
Is just testing create/delete sufficient here?

Reply:
It gets used when a Lustre file system is created, so it's doing more than just a create/delete. I can make some tests for the NnfLustreMGT controller specifically.l